docker-storage-setup.conf

```diff
@@ -1,20 +1,21 @@
-# Specify storage driver one wants to use with docker. Default is devicemapper.
+# Specify storage driver one wants to use with container runtimes.
+# Default is devicemapper.
 # Other possible options are overlay, overlay2 and "". Empty string means do
 # not do any storage setup.
 STORAGE_DRIVER=devicemapper
 
-# A set of extra options that should be appended to the generated
-# DOCKER_STORAGE_OPTIONS string. These options will be passed to the Docker
-# daemon as-is and should be valid Docker storage options.
-# EXTRA_DOCKER_STORAGE_OPTIONS="--storage-opt dm.fs=ext4"
+# Set extra options that will be appended to the generated STORAGE_OPTIONS
```

Comment:
These are just flags in the unit file.  We will just have to modify the UNIT file to take both STORAGE_OPTIONS and DOCKER_STORAGE_OPTIONS.  container-storage-setup will only generate STORAGE_OPTIONS in the furture.